packages/enzyme/src/ReactWrapper.js

```diff
@@ -675,7 +683,7 @@ class ReactWrapper {
    */
   parents(selector) {
     const allParents = this.wrap(
-      this.single('parents', n => parentsOfNode(n, this[ROOT].getNodeInternal())),
+      this.single('parents', n => parentsOfNode(n, this[ROOT][NODE])),
```

Comment:
This seems like it might be a bugfix that should go in separately from an `autoUpdate` option?